man/subscription-manager.8

```diff
@@ -297,87 +274,6 @@ command does two important things. Firstly, it will implicitly remove all of the
 .PP
 This command has no options.
 
-.SS ATTACH OPTIONS
-The
-.B attach
-command applies a specific subscription to the system. This command is not possible to use, when the content access mode of the organization to which the system is registered is simple content access mode.
-
-.TP
-.B --auto
-Automatically attaches the best-matched compatible subscription or subscriptions to the system. This is the default unless
-.B --pool
-or
-.B --file
-are used.
-
-.TP
-.B --pool=POOLID
-Gives the ID for the subscriptions pool (collection of products) to attach to the system. This overrides the default of --auto.
-
-.TP
-.B --file=FILE
-Specifies a file from which to read whitespace-delimited pool IDs. If FILE is "-", the pool IDs will be read from stdin. This overrides the default of
-.B
---auto.
-
-.TP
-.B --quantity=NUMBER
-Attaches a specified number of subscriptions to the system. Subscriptions may have certain limits on them, like the number of sockets on the system or the number of allowed virtual guests. It is possible to attach multiple subscriptions (or
-.I stacking
-subscriptions) to cover the number of sockets, guests, or other characteristics. May not be used with an auto-attach.
-
-
-.TP
-.B --servicelevel=LEVEL
-Sets the preferred service level to use with subscriptions automatically attached to the system. Service levels are commonly premium, standard, and none, though other levels may be available depending on the product and the contract. This option cannot be used when attaching specific pools via
-.B --pool
-or
-.B --file.
-
-.SS AUTO-ATTACH OPTIONS
```

Comment:
ah, note that auto-attach is (still?) not on the removal list -- we'll need to discuss about it